website/docs/integrator-section/verifiable-document/ethereum/wallet.md

```diff
@@ -7,18 +7,16 @@ sidebar_label: Creating a wallet
 The first step that we will need to go through, is the [wallet](/docs/docs-section/appendix/glossary#wallet) creation:
 
 ```bash
-open-attestation wallet create --output-file wallet.json --fund ropsten
+open-attestation wallet create --output-file wallet.json
 ```
 
-> Using `--fund ropsten` will automatically add ethers to your wallet on the ropsten network.
-
 During the creation, you will be prompted for a password. Make sure to remember it for the following steps. You will see a message after completion of the command:
 
 ```text
 ℹ  info      Creating a new wallet
 ? Wallet password [hidden]
 …  awaiting  Encrypting Wallet [====================] [100/100%]
-ℹ  info      [ropsten] Added fund to 0x10cFd56E11e7d66C8d0716Cd2D6B847Cb17ABeeD
+ℹ  info      [goerli] Added fund to 0x10cFd56E11e7d66C8d0716Cd2D6B847Cb17ABeeD
 ℹ  info      Wallet with public address 0x10cFd56E11e7d66C8d0716Cd2D6B847Cb17ABeeD successfully created. Find more details:
 ✔  success   Wallet successfully saved into /home/nebulis/IdeaProjects/open-attestation-cli/wallet.json
```

Comment:
isnt this wrong though? we wont add any funds to the newly created wallet

Reply:
ok haha ya forgot to remove this